app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsDetailsActions/PipelineDetailsActionsButton/index.js

```diff
@@ -345,6 +443,9 @@ export default class PipelineDetailsActionsButton extends Component {
             )}
             <li onClick={this.duplicateConfigAndNavigate}>{T.translate(`${PREFIX}.duplicate`)}</li>
             <li onClick={this.handlePipelineExport}>{T.translate(`${PREFIX}.export`)}</li>
+            <li onClick={this.toggleCommitModal}>
```

Comment:
this should be a button

Reply:
can I just assign button role here? If I change it to button I need to deal with the styles as well

Reply:
are li's focusable? https://benfrain.com/converting-divs-into-accessible-pseudo-buttons/ unsure what exactly needs to be done for li to be accessible - do you need more than role=button?

Also when you do figure it out please add it to the rest of these li's